### PR TITLE
Release the Help Center to 100%

### PR DESCRIPTION
**Pull request by alshakero**
#### Proposed Changes

* This releases the Help Center 100% of users.

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php [hunk 1]
```diff
@@ -383,25 +383,15 @@ function load_tags_education() {
...
 
-	$current_segment = 75; // segment of existing users that will get the help center in %.
-	$user_segment    = get_current_user_id() % 100;
-
-	if ( $is_proxied || $user_segment < $current_segment ) {
-		require_once __DIR__ . '/help-center/class-help-center.php';
-	}
+	require_once __DIR__ . '/help-center/class-help-center.php';
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_help_center' );
```
Comment (arcangelini) [inline apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php:404]: I think you need this to load the Help Center plugin don't you?

**Review by arcangelini — CHANGES_REQUESTED** [review]
I could be wrong but I think you still need the loader.

File: apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php (hunk 1 above)
Comment (alshakero) [reply apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php:404]: Yes ssirr

**Review by arcangelini — APPROVED** [review]
Tested in Local Calypso and everything seems to be working well. Looked over the code again and everything looks good now.

Could be nice to create a followup issue to remove the `shouldShowHelpCenterToUser` function completely.

**Pull request merged**